types/components/messages.d.ts

```diff
@@ -4,6 +4,15 @@ export interface PayPalMessagesComponentOptions {
     style?: {
         layout?: "text" | "flex" | "custom";
         color?: string;
+        logo?: {
+            type?: 'primary' | 'alternative' | 'inline' | 'none';
+            position?: 'left' | 'right' | 'top';
+        };
+        text?: {
+            color?: 'black' | 'white' | 'monochrome' | 'grayscale';
+            size?: 	10 | 11 | 12 | 13 | 14 | 15 | 16;
+            align?: 'left' | 'center' | 'right'
```

Comment:
Also, please can we keep the same double quote format for strings, similar to the `style.layout` values.